assets/test/socket_test.js

```diff
@@ -282,9 +282,9 @@ describe("with transports", done =>{
       socket.connect()
       const spy = sinon.spy(socket.conn, "close")
 
-      socket.disconnect(null, "code", "reason")
+      socket.disconnect(null, 1000, "reason")
 
-      assert(spy.calledWith("code", "reason"))
+      assert(spy.calledWith(1000, "reason"))
```

Comment:
necessary for update to mock-socket.  code must be an integer between certain values or else it fails